src/components/ErrorModal/ErrorModal.tsx

```diff
@@ -0,0 +1,23 @@
+type Props = {
+  errorText?: string;
+  btnAction: () => void;
+};
+
+export const ErrorModal: React.FC<Props> = ({ errorText = '', btnAction }) => {
+  return (
+    <div className="notification is-danger">
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+      <button
+        type="button"
+        className="delete"
+        data-cy="modal-close"
+        onClick={() => btnAction()}
```

Comment:
You can directly reference the 'btnAction' function instead of wrapping it in another function. Replace 'onClick={() => btnAction()}' with 'onClick={btnAction}'.